### PR TITLE
Add a remote data only option, to only run remote data tests.

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: pytest_remotedata/plugin.py [hunk 1]
```diff
@@ -74,6 +82,9 @@ def pytest_runtest_setup(item):
         elif remote_data_config == 'astropy':
             if source == 'any':
                 pytest.skip("need --remote-data option to run")
+    else:
+        if remote_data_only_config:
```
Comment (drdavella) [inline pytest_remotedata/plugin.py]: Looks like this could just be `elif`

File: pytest_remotedata/plugin.py [hunk 2]
```diff
@@ -31,7 +34,11 @@ def pytest_configure(config):
 
     strict_check = bool(config.getini('remote_data_strict'))
 
+    if config.option.remote_data_only and config.option.remote_data != 'any':
+        config.option.remote_data = 'any'
```
Comment (drdavella) [inline pytest_remotedata/plugin.py]: Is this really the behavior we want or do we still want to support narrowing to specific data sources even when running with `--remote-data-only`?

Comment (drdavella) [reply pytest_remotedata/plugin.py]: Also maybe we want to check for invalid user input like providing both `--remote-data=none` and `--remote-data-only`

**Comment by astrofrog** [comment]
I'll close this since @Cadair has mentioned that Sunpy now uses a different approach and I don't think anyone has time to finish this up.

**Comment by bsipocz** [comment]
Nevertheless, this would be a lovely feature, so I'm opening an issue in case anyone in the future would be interested.

#43

**Pull request closed without merging**